### ISSUE TITLE
Migrate to python3

### ISSUE THREAD

**Issue by gabriel-tessier** [jctt1983/HeadsUp#5]
As python 2.7 is supported until January 2020
https://pythonclock.org/

May think about moving the code to python 3.x

**Comment by gabriel-tessier** [comment]
Made the barnch python3 but didn't start yet!

**Comment by jctt1983** [comment]
Good thing you remind me @gabriel-tessier! I was working on this for quite a while and I need to clean a few things before giving you this version. Checkout branch **python36**

I think this version is stable and may go live. If you have time pls check and let me know if you find anything. 

Cheers.

**Comment by gabriel-tessier** [comment]
Maybe better to go with a minimal 3.7 support not with 3.x as some version already reach end of life.

For information about versions end of life:
https://devguide.python.org/#status-of-python-branches

Other resources about new stuff in 3.7 version
https://docs.python.org/3/whatsnew/3.7.html

Or 3.8 so we can use := as explain in this pep
  https://www.python.org/dev/peps/pep-0572/

**Comment by jctt1983** [comment]
Good point, I'll check with Python 3.7... since python 3.6 is reaching the end of life
